persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_24/generated/rainier/generated_client.bal

```diff
@@ -1,34 +1,382 @@
+    }
+
+    isolated resource function get building/[string buildingCode]() returns Building|persist:Error {
+        Building|error result = (check self.persistClients.get(BUILDING).runReadByKeyQuery(Building, buildingCode)).cloneWithType(Building);
+        if result is error {
+            return <persist:Error>error(result.message());
+        }
+        return result;
+    }
+
+    isolated resource function post building(BuildingInsert[] data) returns string[]|persist:Error {
+        _ = check self.persistClients.get(BUILDING).runBatchInsertQuery(data);
+        return from BuildingInsert inserted in data
+            select inserted.buildingCode;
+    }
+
+    isolated resource function put building/[string buildingCode](BuildingUpdate value) returns Building|persist:Error {
+        _ = check self.persistClients.get(BUILDING).runUpdateQuery({"buildingCode": buildingCode}, value);
+        return self->/building/[buildingCode].get();
+    }
+
+    isolated resource function delete building/[string buildingCode]() returns Building|persist:Error {
+        Building 'object = check self->/building/[buildingCode].get();
+        _ = check self.persistClients.get(BUILDING).runDeleteQuery({"buildingCode": buildingCode});
+        return 'object;
+    }
+
+    isolated resource function get department() returns stream<Department, persist:Error?> {
+        stream<record {}, sql:Error?>|persist:Error result = self.persistClients.get(DEPARTMENT).runReadQuery(Department);
+        if result is persist:Error {
+            return new stream<Department, persist:Error?>(new DepartmentStream((), result));
+        } else {
+            return new stream<Department, persist:Error?>(new DepartmentStream(result));
+        }
+    }
+
+    isolated resource function get department/[string deptNo]() returns Department|persist:Error {
+        Department|error result = (check self.persistClients.get(DEPARTMENT).runReadByKeyQuery(Department, deptNo)).cloneWithType(Department);
+        if result is error {
+            return <persist:Error>error(result.message());
+        }
+        return result;
+    }
+
+    isolated resource function post department(DepartmentInsert[] data) returns string[]|persist:Error {
+        _ = check self.persistClients.get(DEPARTMENT).runBatchInsertQuery(data);
+        return from DepartmentInsert inserted in data
+            select inserted.deptNo;
+    }
+
+    isolated resource function put department/[string deptNo](DepartmentUpdate value) returns Department|persist:Error {
+        _ = check self.persistClients.get(DEPARTMENT).runUpdateQuery({"deptNo": deptNo}, value);
+        return self->/department/[deptNo].get();
+    }
+
+    isolated resource function delete department/[string deptNo]() returns Department|persist:Error {
+        Department 'object = check self->/department/[deptNo].get();
+        _ = check self.persistClients.get(DEPARTMENT).runDeleteQuery({"deptNo": deptNo});
+        return 'object;
```

Comment:
```suggestion
        Department result= check self->/department/[deptNo].get();
        _ = check self.persistClients.get(DEPARTMENT).runDeleteQuery({"deptNo": deptNo});
        return result;
```

Lets coordinate with @kaneeldias and change this name if possible

Reply:
@kalaiyarasiganeshalingam shall we fix this

Reply:
Updated in [7e5dedd](https://github.com/ballerina-platform/persist-tools/pull/106/commits/7e5deddedf6e0aa89f1951105e0f2c921725628c)